core/templates/pages/collection-player-page/collection-player-page.component.ts

```diff
@@ -326,14 +326,6 @@ export class CollectionPlayerPageComponent implements OnInit {
     this.whitelistedCollectionIdsForGuestProgress = (
       AppConstants.WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS);
 
-    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
Hm, does this behaviour work as expected if this code is removed?

Reply:
Yes, it is in the ending part of the mobile version of the proof video. Named `fixed.mp4`. Thanks!